integration_tests/support/commands.js

```diff
@@ -1,6 +1,8 @@
 import { exactMatchIgnoreWhitespace, splitIsoDateToParts } from './utils'
 import { recall } from '../fixtures/recall'
 import { getPrisonerResponse, getRecallResponse } from '../mockApis/mockResponses'
+import 'cypress-file-upload'
```

Comment:
This line got moved to support/index.js

Reply:
Ahh, I did wonder about that - came through in the rebase this morning - will remove